roles/openshift_cluster_monitoring_operator/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+openshift_cluster_monitoring_operator_image: quay.io/coreos/cluster-monitoring-operator:198d25d6
+openshift_cluster_monitoring_operator_prometheus_operator_image: quay.io/coreos/prometheus-operator-dev
```

Comment:
these should probably be called "_repo" or "_base_image", as a distinction to the first one, which is actually a full image with tag